report_py3o/README.md

```diff
@@ -0,0 +1,18 @@
+### README ###
+
+
+### LibreOffice Report Engine for Odoo/OpenERP ###
+
+
+This module allows you to convert any type of report into LibreOffice/OpenOffice.
```

Comment:
IMO LibreOffice/OpenOffice is too application specific. This module seems to convert into the [Open Document Format (ODF)](https://en.wikipedia.org/wiki/OpenDocument)